e2e-tests/cypress/support/appeal-submission-appeal-site-address/confirmSiteAddressWasAccepted.js

```diff
@@ -0,0 +1,7 @@
+module.exports = () => {
```

Comment:
Should we also navigate back to the address input page and see that the recently added input values are displayed?

Reply:
Well it's apparently the hole point of the new integration stories, but the back button will not prove that the data is stored. I'll update this PR later as I'm on other tickets now

Reply:
I am not suggesting use of the back button. I would expect navigation by url and to be able to see the newly added data to confirm that it has not been changed in the case of invalid inputs.

Reply:
I did this in the uncommented scenario line as it's the new "template".